js/renovation/inferno_wrapper/__tests__/utils/test_components/inferno_test_widget.tsx

```diff
@@ -54,7 +54,7 @@ export class PreactTestWidgetProps {
   },
   view,
 })
-export default class PreactTestWidget extends JSXComponent(PreactTestWidgetProps) {
+export default class InfernoTestWidget extends JSXComponent(InfernoTestWidgetProps) {
```

Comment:
Can we rename it in something more neutral. RenovationTestWidget, or simple TestWidget